tests/db_engine_specs/mysql_tests.py

```diff
@@ -89,18 +89,10 @@ def test_is_db_column_type_match(self):
             ("TIME", GenericDataType.TEMPORAL),
         )
 
-        for type_expectation in type_expectations:
-            type_str = type_expectation[0]
-            col_type = type_expectation[1]
-            assert MySQLEngineSpec.is_db_column_type_match(
-                type_str, GenericDataType.NUMERIC
-            ) is (col_type == GenericDataType.NUMERIC)
-            assert MySQLEngineSpec.is_db_column_type_match(
-                type_str, GenericDataType.STRING
-            ) is (col_type == GenericDataType.STRING)
-            assert MySQLEngineSpec.is_db_column_type_match(
-                type_str, GenericDataType.TEMPORAL
-            ) is (col_type == GenericDataType.TEMPORAL)
+        for type_str, col_type in type_expectations:
+            print(">>> ", type_str)
```

Comment:
assuming a leftover:
```suggestion
            print(">>> ", type_str)
```